commands/qemu-guest-tools/guesttools.go

```diff
@@ -304,23 +306,25 @@ func (g *guestTools) doExecShell(ID string) {
 	}
 
 	// Create a new shellHandler
-	handler := newShellHandler(ws, g.log.WithField("shell", ID))
+	handler := interactive.NewShellHandler(ws, g.log.WithField("shell", ID))
 
-	// Create a shell
-	shell := exec.Command("sh")
-	shell.Stdin = handler.StdinPipe()
-	shell.Stdout = handler.StdoutPipe()
-	shell.Stderr = handler.StderrPipe()
+	// Set command to standard shell, if no command is given
+	if len(command) == 0 {
+		command = []string{"sh"}
+		if goruntime.GOOS == "windows" {
+			command = []string{"cmd.exe"}
```

Comment:
yeah, maybe... I don't know yet... I don't think we have powershell on say windows xp...
I suspect that as I come further guest-tools will be configured with what shell to use, maybe if people install bash on windows that's an even better option for interactive shells :)